TUTORIAL.md

```diff
@@ -1,438 +0,0 @@
-# Tutorial
```

Comment:
Perhaps leave a link to the IPython tutorial instead of deleting (in case someone links here)